be1-go/message2/messagedata/vote_cast_write_in.go

```diff
@@ -0,0 +1,18 @@
+package messagedata
+
+// VoteCastWriteIn ...
```

Comment:
I don't think this is fully implemented . I'm wondering if we should remove it for the time being, in order to minimize the amount of dead/unsupported/low-priority code.

Reply:
Fully implemented in the rest of the system, I mean.